driver-core/build.gradle

```diff
@@ -52,28 +52,25 @@ buildConfig {
     packageName = 'com.mongodb.internal.build'
 }
 
-jar {
-    bnd(
-            '-exportcontents': "*;-noimport:=true",
-            'Automatic-Module-Name': 'org.mongodb.driver.core',
-            'Build-Version': project.gitVersion,
-            'Import-Package': [
-                    'org.bson.*',                       // unfortunate that this is necessary, but if it's left out then it's not included
-                    'javax.crypto.*',
-                    'javax.crypto.spec.*',
-                    'javax.management.*',
-                    'javax.naming.*',
-                    'javax.naming.directory.*',
-                    'javax.net.*',
-                    'javax.net.ssl.*',
-                    'javax.security.sasl.*',
-                    'javax.security.auth.callback.*',
-                    'org.ietf.jgss.*',
-                    'io.netty.*;resolution:=optional',
-                    'org.xerial.snappy.*;resolution:=optional',
-                    'com.github.luben.zstd.*;resolution:=optional',
-                    'org.slf4j;resolution:=optional',
-                    'jnr.unixsocket;resolution:=optional'
-            ].join(',')
-    )
+afterEvaluate {
```

Comment:
An example of overriding manifest attributes. `Automatic-Module-Name` and `Bundle-SymbolicName` have different values than what the default would generate.

Reply:
Good catch & work around.